### PR TITLE
maintenance, tox, github actions

### PR DESCRIPTION
**Pull request by JakaKokosar**
https://github.com/biolab/orange3/issues/5145

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -65,14 +69,14 @@
...
     global DATA_FILES
-    if "bdist_wheel" in sys.argv and not path.exists(local_dir):
-        print(
-            "Directory '{}' does not exist. "
-            "Please build documentation before running bdist_wheel.".format(
-                path.abspath(local_dir)
-            )
-        )
-        sys.exit(0)
+    # if "bdist_wheel" in sys.argv and not path.exists(local_dir):
```
Comment (PrimozGodec) [inline setup.py:72]: @JakaKokosar were there problems with tests that you commented this out? What exactly is the reason for it?

Comment (JakaKokosar) [reply setup.py:72]: Yeah, it breaks building package when running tox. This is just a temporary workaround. When we finish migrating from Travis to GH actions/tox we will make sure that building docs will be a part of a add-on build process (similar to what Aleš has done with Orange).

Comment (PrimozGodec) [reply setup.py:72]: Ok, great.

**Comment by PrimozGodec** [comment]
It looks good.

**Pull request merged**